app/carbonapi/app.go

```diff
@@ -101,12 +101,14 @@ func New(config cfg.API, logger *zap.Logger, buildVersion string) (*App, error)
 // Start starts the app: inits handlers, logger, starts HTTP server
 func (app *App) Start() func() {
 	logger := zapwriter.Logger("carbonapi")
+	accessLogger := zapwriter.Logger("access")
```

Comment:
Does this actually reduce the number of loggers and move us clser to having a single logger?

Reply:
Or is this maybe rerquired as a preliminary step?

Reply:
With the latest changes we have 3 loggers:
* main: stdout logger in the main
* access: access logging
* app (carbonapi/zipper): setup, handlers

We can merge app and access but I think it's reasonable to have them separated.

Reply:
I am pretty sure we need just *one* logger. I do not see why we'd need more.

Reply:
That is the strategy of majority of projects. It adds uniformity and clarity. There should be a good reason to have more than one.

Reply:
At the current state, loggers are controlled. Both loggers are initialized on app.Start and passed through the handlers. No magic, no global logger, etc.
The one in the main can be eliminated by using "log" instead of "zap". But I don't think that is a big deal.
For the other two, we have the separation in the code. Access logs are separated from logic logs and they can be configured and used differently.
If we merge this two, we do not get much benefit IMO.

Reply:
What benefit do we get from having two loggers? Why not have 10?
For me, I don't understand why we should have a separate access log. Neither is it clear when should I use it.

If we do not need two loggers, added complexity is not necessary. I think that should be obvious. This seems like a wrong design decision to me that is the leftovers of the problem that we are trying to solve.